xml/System.Text.Json/JsonElement+ArrayEnumerator.xml

```diff
@@ -196,8 +196,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.Generic.IEnumerator" /> object that can be used to iterate through the collection.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.Json.JsonElement.ArrayEnumerator> instance is cast to an <xref:System.Collections.IEnumerable> interface. 

         ]]></format>
         </remarks>
```